From fb38f321dc684bc3b576932e8303738af6174476 Mon Sep 17 00:00:00 2001
From: Guan Jing <guanjing6@huawei.com>
Date: Wed, 30 Nov 2022 18:23:25 +0800
Subject: [PATCH 081/500] [Huawei] sched/fair: Fix kabi borken in
 sched_domain_shared

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 188420

--------------------------------

The sched_domain_shared structure is only used as pointer, and other
drivers don't use it directly.

Signed-off-by: Guan Jing <guanjing6@huawei.com>
---
 include/linux/sched/topology.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/sched/topology.h b/include/linux/sched/topology.h
index fe4e222d5d2d..249c98aef083 100644
--- a/include/linux/sched/topology.h
+++ b/include/linux/sched/topology.h
@@ -82,10 +82,10 @@ struct sched_domain_shared {
 	atomic_t	ref;
 	atomic_t	nr_busy_cpus;
 	int		has_idle_cores;
-	int		nr_idle_scan;
 #ifdef CONFIG_SCHED_STEAL
 	struct sparsemask *cfs_overload_cpus;
 #endif
+	KABI_EXTEND(int		nr_idle_scan)
 };
 
 struct sched_domain {
-- 
2.17.1

